package.json

```diff
@@ -74,16 +74,20 @@
     "webpack-dev-server": "^2.5.0"
   },
   "dependencies": {
-    "pdfobject": "2.0.201604172",
+    "babel-runtime": "6.23.0",
     "blueimp-load-image": "2.12.2",
     "classnames": "2.2.5",
     "countup.js": "1.8.1",
+    "deep-equal": "1.0.1",
     "enumerate-devices": "^1.1.0",
+    "eventemitter2": "2.1.3",
     "history": "4.5.1",
     "object-loops": "0.8.0",
     "parse-unit": "1.0.1",
-    "preact": "6.0.2",
```

Comment:
That's fine - just making sure it wasn't being squeezed in as a "minor" change 😆